src/include/robin_hood.h

```diff
@@ -971,9 +970,9 @@ class unordered_map
             do {
                 auto const n = detail::unaligned_load<size_t>(mInfo);
 #if ROBIN_HOOD_LITTLE_ENDIAN
-                inc = ROBIN_HOOD_COUNT_TRAILING_ZEROES(n) / 8;
```

Comment:
I don't such a change is worth it. There are many constants like this all over the code

Reply:
That’s a good point. There are a handful of “magic numbers”, not all of them easily defined. It’s more a stylistic thing. I considered this a first step to defining all of them. But I’m not wed to the change; I can revert it if you’d prefer.